.travis.yml

```diff
@@ -4,7 +4,7 @@ language: php
 
 branches:
   except:
-    - /^release-.*$/
+    - /^release-\d+\.\d+\.\d+.*$/
     - /^ghgfk-.*$/
```

Comment:
ghgfk is a webtool for to do the git flow merge. For work it needs a temporal branch which is immediately removed so Travis error when try to checkout that branch